packages/ng/form-field/form-field.component.ts

```diff
@@ -0,0 +1,125 @@
+import { AfterViewInit, booleanAttribute, Component, ContentChild, HostBinding, inject, Input, OnChanges, OnDestroy } from '@angular/core';
+import { NgClass, NgIf, NgSwitch, NgSwitchCase, NgTemplateOutlet } from '@angular/common';
+import { InputDirective } from './input.directive';
+import { FormFieldSize } from './form-field-size';
+import { BehaviorSubject } from 'rxjs';
+import { InlineMessageComponent, InlineMessageState } from '@lucca-front/ng/inline-message';
+import { SafeHtml } from '@angular/platform-browser';
+import { LuTooltipModule } from '@lucca-front/ng/tooltip';
+import { ReactiveFormsModule } from '@angular/forms';
+
+let nextId = 0;
+
+@Component({
+	selector: 'lu-form-field',
+	standalone: true,
+	imports: [NgIf, NgSwitch, NgSwitchCase, NgTemplateOutlet, InlineMessageComponent, LuTooltipModule, ReactiveFormsModule],
+	templateUrl: './form-field.component.html',
+	styleUrls: ['./form-field.component.scss'],
+	hostDirectives: [NgClass],
```

Comment:
Using this prevent the consumer of using `<lu-form-field [ngClass]="..." />`.

One simple trick is to create a copy of the NgClass:

```ts
@Directive({
	selector: '[ngClazz]',
	standalone: true,
})
class NgClazz extends NgClass {}
```

```suggestion
	hostDirectives: [NgClazz],
```